ihatemoney/web.py

```diff
@@ -363,12 +363,24 @@ def remind_password():
             if success:
                 return redirect(url_for(".password_reminder_sent"))
             else:
+                # Only display admin email if its not the default and public creation is allowed
+                admin_email = current_app.config.get("MAIL_DEFAULT_SENDER")
+                error_extension = ""
+                if (
+                    admin_email
+                    and admin_email[1] != "budget@notmyidea.org"
```

Comment:
This makes sense. This sounds like I should change `MAIL_DEFAULT_SENDER` throughout the repository to a generic admin@email.com or something along those lines?

Reply:
Yes that seems good !

Reply:
I can't find your pull request, but I thought you did, right? I've read about using `example.com` rather than other domain names today, see the rationale here : https://news.ycombinator.com/item?id=29623646

Reply:
Thanks for bringing this up, I didn't know there was a difference. Changes made.